Modules/SharePointDsc/DSCResources/MSFT_SPUserProfileSyncService/readme.md

```diff
@@ -4,15 +4,25 @@ This resource is responsible for ensuring that the user profile sync service
 has been provisioned (Ensure = "Present") or is not running (Ensure =
 "Absent") on the current server.
 
-This resource requires that the FarmAccount is specified as the InstallAccount
-or PsDscRunAsCredential parameter. It will throw an exception if this is not
-the case.
+The specified InstallAccount or PSDSCRunAsCredential shouldn't be the Farm Account.
+The resource will throw an error when it is. However, the FarmAccount parameter
+should be the Farm Account. The resource will throw an error if it is not. This is
```

Comment:
Another place that should be reworded since the Farm account will be pulled from managed accounts and the param deprecated. Also, in the "Note:" section below.

Reply:
Since I initially added the FarmAccount parameter and later on replaced this with retrieving the account from the Managed Accounts, I forgot to update the Readme file